src/index.html

```diff
@@ -4,9 +4,567 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com" crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a href="#home" class="logo">
+          <img src="/images/logo.png" alt="logo">
+        </a>
+        <nav class="nav">
+          <ul class="nav__list-items">
+            <li class="nav__items">
+              <a
+                href="#apple"
+                class="nav__link is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__items">
+              <a href="#samsung" class="nav__link">Samsung</a>
+            </li>
+            <li class="nav__items">
+              <a href="#smartphones" class="nav__link">Smartphones</a>
+            </li>
+            <li class="nav__items">
+              <a
+                href="#laptops"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__items">
+              <a href="#gadgets" class="nav__link">Gadgets</a>
+            </li>
+            <li class="nav__items">
+              <a href="#tablets" class="nav__link">Tablets</a>
+            </li>
+            <li class="nav__items">
+              <a href="#photo" class="nav__link">Photo</a>
+            </li>
+            <li class="nav__items">
+              <a href="#video" class="nav__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="main">
+      <div class="container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__img"
+            src="../src/images/imac.jpeg"
+            alt="Apple laptop"
+          >
+
+          <h1 class="card__title">
+            APPLE A1419 iMac 27" Retina
+            <br>
+            5K Monoblock (MNED2UA/A)
+          </h1>
+
+          <p class="card__code">
+              Product code: 195434
+          </p>
+
+          <div class="stars">
+            <div class="stars__icon">
+              <div
+                class="stars__review"
+              >
+              </div>
+              <div
+                class="stars__review"
+              >
```

Comment:
When there are only 1 or 2 attributes, then you do not need to move them to a new line, you need to do this only when there are 3 or more attributes, you need to fix it everywhere
```suggestion
              <div class="stars__review">
```